### DISCUSSION TITLE
Profile URL for update_profile

### DISCUSSION THREAD

**Discussion by duber000** [fief-dev/fief#48] [Q&A] [answered]
Hello, I am testing updating a user_profile with a custom field

`
...
   access_token_info: FiefAccessTokenInfo = Depends(auth.authenticated()),
):
    await fief.update_profile(access_token_info["access_token"], {"fields": {"pluscode": "859665RJ+PD"}})
`

I get an error 404 Not Found for url 'https://redacted.fief.dev/profile' 

Is there a different way I should be attempting this?

**Comment by frankie567** [comment, marked as answer]
Whoops, actually this is a mistake in the Python client, sorry 😅  I'll fix this right away!

**Reply by frankie567** [reply]
Fix released in [v0.13.1](https://github.com/fief-dev/fief-python/releases/tag/v0.13.1).
